src/utils/validateInput.js

```diff
@@ -0,0 +1,30 @@
+function validateInput(originalText, targetCase) {
+  const validationMessages = [];
+
+  if (!targetCase) {
+    validationMessages.push({
+      message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (!originalText.trim()) {
+    validationMessages.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (
+    targetCase &&
+    !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].some(
+      (caseType) => caseType === targetCase.toUpperCase(),
+    )
+  ) {
+    validationMessages.push({
+      message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
```

Comment:
Create a constant for `['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER']`, and use it to create a message with available cases